code/index.html

```diff
@@ -1,22 +1,103 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./style.css" />
+  <title>Project 2: The News Site</title>
+</head>
+
+<body>
+  <header>
+    <div class="site-name">
+      <img class="logo" src="heartbeat.svg" alt="Logo">
+      <h1>Health Daily</h1>
+    </div>
+    <nav>
+      <a href="" target="_blank">Womens health</a>
+      <a href="" target="_blank">Mens health</a>
+      <img class="nav-logo" src="heartbeat.svg" alt="Logo">
+      <a href="" target="_blank">Childrens health</a>
+      <a href="" target="_blank">About us</a>
+    </nav>
+  </header>
+
+  <section class="big-news">
+    <img src="hans-reniers-lQGJCMY5qcM-unsplash.jpg" alt="Image of glass containers">
```

Comment:
You actually don't need to add "image" to the alt. It will read "image + alt text" so now it will be "image image of glass container"